specs/MultiProfile.md

```diff
@@ -328,6 +362,15 @@ interface ICoreWebView2Profile2 : ICoreWebView2Profile {
   /// See ICoreWebView2CookieManager.
   [propget] HRESULT CookieManager([out, retval] ICoreWebView2CookieManager** cookieManager);
 }
+
+[uuid(1c1ae2cc-d5c2-ffe3-d3e7-7857035d23b7), object, pointer_default(unique)]
+interface ICoreWebView2Profile3 : ICoreWebView2Profile2 {
+  /// All webviews on this profile will be closed, and the profile will be marked for deletion.
```

Comment:
Its not clear to me reading this description which things will happen during the call to Delete and which will happen later. If these things are the only things that happen during the Delete call, then I would add a line like this:

```suggestion
  /// All webviews on this profile will be closed, and the profile will be marked for deletion.
  /// After the Delete() call completes, the render ...
```